app/templates/account/profile.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+	<div> {{ user.first_name }} </div>
+    <div> {{ user.last_name }} </div>
+    <div> {{ user.email }} </div>
+    <div> {{ user.role.name }} </div>
+    <div> {{ user.birthday }} </div>
+    <div>
+		{% if is_current %}
```

Comment:
this whole section still looks like this to me
<img width="526" alt="screen shot 2015-11-17 at 3 59 28 pm" src="https://cloud.githubusercontent.com/assets/3924168/11224693/3ff2ea48-8d44-11e5-9e9c-6b097d19f59a.png">
it should look like this
```html
{% block content %}
    <div>{{ user.first_name }}</div>
    <div>{{ user.last_name }}</div>
    <div>{{ user.email }}</div>
    <div>{{ user.role.name }}</div>
    <div>{{ user.birthday }}</div>
    <div>
    {% if is_current %}
        <div>
            <a href="{{ url_for('account.edit_profile') }}">
                <button class="ui button">Edit</button>
            </a>
        </div>
    {% endif %}
{% endblock %}
```